clients/ts/signalr/src/Utils.ts

```diff
@@ -15,3 +18,52 @@ export class Arg {
         }
     }
 }
+
+export function getDataDetail(data: any, includeContent: boolean): string {
+    let length: string = null;
+    if (data instanceof ArrayBuffer) {
+        length = `Binary data of length ${data.byteLength}.`;
+        if (includeContent) {
+            length += ` Content: '${formatArrayBuffer(data)}'.`;
```

Comment:
Could we remove the '.' from here, logs are now showing up with '..' at the end because the actual log location also puts a '.'